core/server/master/src/main/java/alluxio/master/file/meta/LockingScheme.java

```diff
@@ -86,6 +92,10 @@ public LockPattern getPattern() {
   }
 
   /**
+   * Gets the path for this locking scheme.
```

Comment:
I believe that `Gets the path for this locking scheme and return it` is better. What do you think?